mappings/net/minecraft/network/listener/ServerPlayPacketListener.mapping

```diff
@@ -17,7 +17,7 @@ CLASS net/minecraft/class_2792 net/minecraft/network/listener/ServerPlayPacketLi
 		ARG 1 packet
 	METHOD method_12053 onBookUpdate (Lnet/minecraft/class_2820;)V
 		ARG 1 packet
-	METHOD method_12054 onGuiClose (Lnet/minecraft/class_2815;)V
+	METHOD method_12054 onCloseMenu (Lnet/minecraft/class_2815;)V
```

Comment:
```suggestion
	METHOD method_12054 onMenuClose (Lnet/minecraft/class_2815;)V
```
imo the packet doesnt' need renaming but this method name is more congruent with other methods.